tools/c7n_org/scripts/orgaccounts.py

```diff
@@ -37,7 +37,9 @@
     '-f', '--output', type=click.File('w'),
     help="File to store the generated config (default stdout)")
 @click.option('-a', '--active', default=False, help="Get only active accounts", type=click.BOOL)
-def main(role, ou, assume, profile, output, regions, active):
+@click.option('-ex', '--exceptions', multiple=True,
```

Comment:
how about renaming this to skip or ignore, exceptions is a pretty nebulous term, especially in the context of policy management/execution.

Reply:
Better, indeed! Changed!